packages/fxa-content-server/app/scripts/templates/pair/index.mustache

```diff
@@ -77,7 +77,7 @@
     {{/askMobileStatus}}
 
     {{^askMobileStatus}}
-      <div class="{{graphicId}} mt-5" role="img" aria-label="{{#t}}Complete your set-up{{/t}}"></div>
+      <div class="{{graphicId}} mt-5" role="img" aria-label="{{#t}}A tablet, a laptop, and a smartphone with hearts on their screens{{/t}}"></div>
```

Comment:
A11y's original recommendation was to make screen readers ignore this image since it's decorative, but I asked for clarification since it took up so much screen real estate and it seemed like someone with low vision could see something here and wonder what it was. They agreed and recommended this alt text, and gave another scenario of someone explaining to their grandmother to "click the button below the image with devices" or something similar which re-enforced the need to have this be read to better match what a sighted user would see and think.